chef/data_bags/crowbar/bc-template-provisioner.schema

```diff
@@ -49,6 +49,22 @@
                   "required": false,
                   "mapping": {
                     "ssh_password": { "type": "str", "required": false },
+                    "packages": {
+                      "type": "map",
+                      "required": false,
+                      "mapping": {
+                        "suse-11_3": {
+                          "type": "seq",
+                          "required": false,
+                          "sequence": [ { "type": "str" } ]
+                        },
+                        "suse-12_0": {
```

Comment:
Btw, we should definitely not hard-code the OS for which this work in the schema. See how it's done for the `repos` attribute a few lines after this.